torchaudio/sox_effects/__init__.py

```diff
@@ -1,4 +1,4 @@
-from torchaudio._internal import module_utils as _mod_utils
+from torchaudio._internal.module_utils import is_sox_available
```

Comment:
The reason why we adopted the import pattern of `from torchaudio._internal import module_utils as _mod_utils` is that the whole module (and functions inside of it) is intended for internal use, so we do not want to expose it in other module.

`from torchaudio._internal.module_utils import is_sox_available` makes the helper function accessible like `torchaudio.XXX.is_sox_available`. Of course this is not a documented function and it should not be used by users, but we want to be careful about it. This is because at the beginning of torchaudio project, many helper functions were defined in `__init__.py` without under score prefix and it was messy/confusing. If we import individual functions from internal module, users cannot tell from the outside of the package that the function resides in the internal module, thus not for users to use.

Reply:
Of course, you can do `from torchaudio._internal.module_utils import is_sox_available as _is_sox_available` too.